From 63ef7628091c8827e3d0d688220d3ae165587eb2 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Cristi=C3=A1n=20Maureira-Fredes?=
 <cristian.maureira-fredes@qt.io>
Date: Thu, 13 Jul 2023 14:41:14 +0200
Subject: adapt qC{Debug,Info,Warning,Critical} for new definition

In response to:
   I3eb1bd30e0124f89a052fffd16a7564f4450ec8a

Change-Id: I5393660521a8ee12a14bf2c287a6ea9a0393408f
Reviewed-by: Shyamnath Premnadh <Shyamnath.Premnadh@qt.io>
Reviewed-by: Qt CI Bot <qt_ci_bot@qt-project.org>
---
 sources/pyside6/PySide6/QtCore/typesystem_core_common.xml | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/sources/pyside6/PySide6/QtCore/typesystem_core_common.xml b/sources/pyside6/PySide6/QtCore/typesystem_core_common.xml
index bf6e263d4..e8155d41d 100644
--- a/sources/pyside6/PySide6/QtCore/typesystem_core_common.xml
+++ b/sources/pyside6/PySide6/QtCore/typesystem_core_common.xml
@@ -3340,25 +3340,25 @@
     <include file-name="qloggingcategory.h" location="global" />
     </extra-includes>
     <inject-code class="target" position="beginning" file="../glue/qtcore.cpp" snippet="qloggingcategory_to_cpp">
-      qCDebug(*category, "%s", %2);
+      qCDebug((*category), "%s", %2);
     </inject-code>
   </add-function>
 
   <add-function signature="qCCritical(PyObject *, const char *)">
     <inject-code class="target" position="beginning" file="../glue/qtcore.cpp" snippet="qloggingcategory_to_cpp">
-      qCCritical(*category, "%s", %2);
+      qCCritical((*category), "%s", %2);
     </inject-code>
   </add-function>
 
   <add-function signature="qCInfo(PyObject *, const char *)">
     <inject-code class="target" position="beginning" file="../glue/qtcore.cpp" snippet="qloggingcategory_to_cpp">
-      qCInfo(*category, "%s", %2);
+      qCInfo((*category), "%s", %2);
     </inject-code>
   </add-function>
 
   <add-function signature="qCWarning(PyObject *, const char *)">
     <inject-code class="target" position="beginning" file="../glue/qtcore.cpp" snippet="qloggingcategory_to_cpp">
-      qCWarning(*category, "%s", %2);
+      qCWarning((*category), "%s", %2);
     </inject-code>
   </add-function>
 
-- 
cgit v1.2.3

